cmd/snap/cmd_info.go

```diff
@@ -462,17 +434,32 @@ func (x *infoCmd) Execute([]string) error {
 			continue
 		}
 
-		if tryDirect(w, snapName, x.Verbose, termWidth) {
-			noneOK = false
-			continue
-		}
-		remote, resInfo, _ := x.client.FindOne(snapName)
-		local, _, _ := x.client.Snap(snapName)
+		var direct, local, remote *client.Snap
+		var resInfo *client.ResultInfo
+		var directErr, remoteErr, localErr error
 
-		both := coalesce(local, remote)
+		direct, directErr = loadDirect(snapName)
+		if directErr != nil {
+			remote, resInfo, remoteErr = x.client.FindOne(snapName)
+			local, _, localErr = x.client.Snap(snapName)
+		}
+		// note direct == nil, or local == nil and remote == nil
+		theSnap := coalesce(direct, local, remote)
 
-		if both == nil {
+		if theSnap == nil {
+			if x.Verbose {
+				if directErr != nil {
```

Comment:
if we get here, we will always print the 3 no? not a fan of the stacked error error error output

Reply:
yeah, i'm open to ideas on this front

Reply:
what are we trying to address here, are the error content usually meaningful here?  is the problem that not found is too generic?  I'm also not sure that the first impulse if something doesn't work is to switch to verbose? Maybe we should always pack more info in the actual error we return

Reply:
I'll leave out this bit of the change for now. The intention is that as we 'hide' the error, people will get 'not found' for info of a remote snap when the network is out for example, which is suboptimal. But in those cases this is too verbose also. So it needs more thought anyway.

Reply:
if the goal is to land it quickly we should undo this bit for now then